addon/index.js

```diff
@@ -359,6 +361,15 @@ function installExperiment(experiment) {
   }
 }
 
+function uninstallSelf() {
+  // First, kick out all the experiment add-ons
+  Object.keys(store.installedAddons).forEach(id => {
+    uninstallExperiment({addon_id: id});
+  });
```

Comment:
Maybe we should pull this and https://github.com/mozilla/testpilot/blob/master/addon/index.js#L155 into an `uninstallAll` function.